net/linuxptp/patches/020-sad-allow-disabling-MAC-library-autodetection.patch

```diff
@@ -0,0 +1,193 @@
+From 41ed39e7abbdcc68d9ff4baaf022aa2e6d9dadde Mon Sep 17 00:00:00 2001
+From: Shenghao Yang <me@shenghaoyang.info>
+Date: Sun, 15 Sep 2024 14:53:21 +0800
+Subject: [PATCH] sad: allow disabling MAC library autodetection
+
+Signed-off-by: Shenghao Yang <me@shenghaoyang.info>
```

Comment:
Shouldnt be here included commit description from the patch, which you sent to upstream?
https://lists.nwtime.org/sympa/arc/linuxptp-devel/2024-09/msg00051.html

Reply:
Ah indeed, used an initial one that didn't have a commit message - let me refresh that patch.